js/scripts/dimension/playground_dimension.js

```diff
@@ -91,7 +91,7 @@ const loadPlaygroundDimension = () => {
         // NOTE: we can't activate music until the user has interacted with the canvas. (this issue is inherent to HTML5)
```

Comment:
Lines 91-101 look like they are duplicates of lines 79-89.

Reply:
You're right. This wasn't my class, this was something that I was fixing to get rid of a red error bar in the firefox debugger.